src/components/HomeReadMore.js

```diff
@@ -0,0 +1,56 @@
+import get from 'lodash/get';
```

Comment:
Created a new file so I wouldn't touch the old ReadMore which is still used on another page. The original ReadMore expects pure text, and outputs `...Read More` in-line. I could spend more time to combine the two (which is probably ideal, design-wise).

Reply:
If you want to take a stab at merging those files, that would be ideal, but I'm fine either way.